dags/common/enhance.py

```diff
@@ -0,0 +1,54 @@
+import datetime
+
+
+class Enhancement:
+    def __init__(self):
+        self.creation_date = datetime.datetime.now().isoformat()
+
+    def constuct_abstracts(self, item, publisher):
```

Comment:
These functions are only used in this class, thus we should add a leading underscore in the name, as described in [PEP 8](https://peps.python.org/pep-0008/)

> Use one leading underscore only for non-public methods and instance variables.